winforms/formatting-utility/vb/Form1.vb

```diff
@@ -7,20 +7,20 @@ Public Class Form1
 
    Private label As ToolStripStatusLabel
 
-    Private ReadOnly rm As New ResourceManager("Formatter.Resources", Me.GetType().Assembly)
+    Private ReadOnly rm As New ResourceManager("Formatter.Resources", [GetType]().Assembly)
```

Comment:
This change is debatable, I prefer Me others [GetType]. VS doesn't give the choice to use Me here but not prefer it everywhere.

Reply:
I would agree that Me.GetType is preferable here.

Reply:
fixed